src/styles/main-container.scss

```diff
@@ -0,0 +1,14 @@
+.main {
+  display: flex;
+  justify-content: center;
+  box-sizing: border-box;
+
+  &__container {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    box-sizing: border-box;
+    width: 950px;
+    margin: 50px 40px;
```

Comment:
in mockup we can see that 1024 and 1200 have different space (margins/paddings) between border of the page and a card 
i thing it should change

Reply:
Thanks for review, but i think that  spaces between border and products should be changeble because of fixed spaces between cards and fixed cards container width 
:question:

Reply:
point from check list
13 [LAYOUT] - don't hardcode max-width, your catalog container should work fine for any number of cards
(here the is container width should not be fixed. )

there what i talking about. Different distance 
https://prnt.sc/vl02QLHDVACC for 1024px
https://prnt.sc/dv1abmdmGQAI for 1200px

Reply:
i think i fixed this))

Reply:
Nice GJ! 
But it still breaks when screen is small. But don't worry you will fix this in future HW.
i think for this task it is ok))) lets see what is mentors will say

Reply:
it doesn't break, i just want it to transfer on the next row if the screen is smaller than the width of 4 cards plus paddings
Soooooo let see)))
I think only mentor can answer)))
But thank you for your attention !)